docs/features/shared-access-key.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Authentication with shared access keys via ASP.NET Core authentication filters"
+layout: default
+---
+
+## Arcus.WebApi.Security.SharedAccessKeyAuthentication
+
+The `Arcus.WebApi.Security` package provides an mechanism that uses shared access keys to authenticate users.
+This authentication process consists of two parts:
+
+1. Looks for the configured HTTP request header that contains the shared access key
+2. Matches this value with the configured **Key Vault** secret
```

Comment:
Actually, this is not true, we will look in whatever secret provider is configured